src/components/pages/about-us/team.tsx

```diff
@@ -1,51 +1,109 @@
-import React from 'react';
-import { useTranslation } from 'gatsby-plugin-react-i18next';
+import React, { ReactNode } from 'react';
 import styled from 'styled-components';
-import { up } from '../../support/breakpoint';
-import { Image } from '../../ui/image/image';
-import { SectionHeader } from '../../content/section-header/section-header';
-import { SyTeamMember } from '../../../types';
+import { down } from '../../support/breakpoint';
+import { GalleryItem } from '../../../types';
 import { GatsbyImage, getImage } from 'gatsby-plugin-image';
 
-const TeamLayout = styled.div`
-  margin-top: 80px;
+// the number of vertical tiles
+const rowCount = 38;
 
+const TeamLayout = styled.div`
   display: grid;
+  overflow: hidden;
+  object-fit: cover;
+  // atomical tile height: 250px,
+  grid-template-rows: repeat(${rowCount}, 250px);
+  grid-template-columns: repeat(2, calc(50% - 12px));
+  align-items: stretch;
+  justify-items: stretch;
+  gap: 24px;
+  grid-auto-flow: row;
+  // combine multiple smaller tiles into bigger ones
+  // each line represents one row, each word ('pN') represents one column
+  grid-template-areas:
```

Comment:
Interesting. Can you say a few more words about this solution? It's pragmatic, which I love for this case But is this your solution? Did you find it somewhere? How will it overflow with more images? What's the current capacity?

Reply:
The solution sets up a CSS grid with multiple custom-defined grid areas so that the images can be placed exactly where the design stipulates. The current capacity is 38 rows of atomic tiles. When new images are added to the bottom, any overflowing elements will only be included if there is an area defined for them. Since the mobile version does not use grids, the overflowing image will still be shown on mobile. I realized that this is not optimal and I will try to fix this. Inspired by the CSS grids I used on the BadgeForge project, I came up with this grid-area solution. At the moment it is rather inflexible, so I will look into making editing the grid easier.

Reply:
As of 66978e4620155aedf04e77378599cf3c9082e863, the tile size can be set directly in Contentful for each image. This removes the need for predefined grid areas, and all overflowing items will now be placed automatically according to their tile size.